Pipeline/CreateUCDComponentVersion/dbb-ucd-packaging.groovy

```diff
@@ -125,7 +129,14 @@ properties.buildReportOrder.each{ buildReportFile ->
 		}
 	}
 
+    def USSRecords = buildReport.getRecords().findAll{
```

Comment:
`ussRecords` would better align how we use variables within the script. If starting with upper case, it feels like a type of an object. 

Maybe we need to update the `executes` also to `executeRecords` to make it easier to read.

Reply:
Done